pkg/repoconfig/merge/combine.go

```diff
@@ -1,12 +1,12 @@
 package merge
 
 import (
-	"github.com/jenkins-x/lighthouse/pkg/apis/lighthouse/v1alpha1"
 	"github.com/jenkins-x/lighthouse/pkg/config"
+	"github.com/jenkins-x/lighthouse/pkg/repoconfig"
```

Comment:
so we go from generated types to fixed ones? Are we fixing a regression here or what is actually happening here? So `RepositoryConfig` never meant to be in the api package?

Reply:
Yeah, it shouldn't have ever been in the api package. That was a screw-up on my part asking for that. Pretty much everything around `repoconfig` is unfinished/in need of revisiting, so I just thought it'd be best to get it out of `pkg/apis` for now.

Reply:
> Yeah, it shouldn't have ever been in the api package. That was a screw-up on my part asking for that. Pretty much everything around repoconfig is unfinished/in need of revisiting, so I just thought it'd be best to get it out of pkg/apis for now.

Ok, got you. Thanks for the clarification.